app/models/category.rb

```diff
@@ -1,2 +1,5 @@
 class Category < ApplicationRecord
+  validates :name, presence: true, uniqueness: true, length: { maximum: 255, too_long: "is too long (maximum is 255 characters)" }
```

Comment:
ko nên custom message tại file này, nếu có custom thì sửa trong file yml

Reply:
tham khảo: https://guides.rubyonrails.org/v3.2/i18n.html#translations-for-active-record-models